src/components/VenueDetails/BookingCard.jsx

```diff
@@ -0,0 +1,171 @@
+import {
+  Box,
+  Typography,
+  Card,
+  Button,
+  Checkbox,
+  FormControlLabel,
+  Divider,
+} from '@mui/material';
+import { DatePicker } from '@mui/x-date-pickers/DatePicker';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { useState } from 'react';
+import { differenceInDays } from 'date-fns';
+import { useExchangeRate } from '../../hooks/useExchangeRate';
+
+export const BookingCard = ({ venue }) => {
```

Comment:
This component is very long. Create custom hooks and separate, smaller components.